src/util/compilationDatabase.ml

```diff
@@ -87,8 +76,8 @@ let load_and_preprocess ~all_cppflags filename =
       let cwd = reroot obj.directory in
       if GobConfig.get_bool "dbg.verbose" then
         Printf.printf "Preprocessing %s\n  to %s\n  using %s\n  in %s\n" file preprocessed_file preprocess_command cwd;
-      system ~cwd preprocess_command; (* command/arguments might have paths relative to directory *)
-      Some preprocessed_file
+      let preprocess_task = {ProcessPool.command = preprocess_command; cwd = Some cwd} in (* command/arguments might have paths relative to directory *)
```

Comment:
Is prefixing `command` with `ProcessPool` necessary here for the correct type to be inferred?

Reply:
Yes, because the `command_object` type in this module also has a `command` field.